examples/driver/huawei_oceanstor_block.yaml

```diff
@@ -5,18 +5,20 @@ authOptions:
   EnableEncrypted: false
   # Encryption and decryption tool. Default value is aes. The decryption tool can only decrypt the corresponding ciphertext.
   PwdEncrypter: "aes"
-  uri: "https://0.0.0.0:8088/deviceManager/rest"
+  endpoints: "https://0.0.0.0:8088/deviceManager/rest"
 
 pool:
-  opensds_file:
-    storageType: file
-    availabilityZone: fileshare
+  opensds_block:
+    storageType: block
+    availabilityZone: default
     extras:
       dataStorage:
-        storageAccessCapability: ["Read","Write","Execute"]          
+        provisioningPolicy: Thin
+        isSpaceEfficient: false
```

Comment:
Can you please confirm if Hotpot still supports isSpaceEfficient

Reply:
please change isSpaceEfficient to 
```
        compression: false
        deduplication: false
```

Reply:
Done